src/main/webapp/i18n/en/user-management.json

```diff
@@ -1,5 +1,13 @@
 {
     "userManagement": {
+        "batch": {
+            "all": "All",
+            "delete": {
+                "question": "Are you sure you want to permanently delete <strong>{{ title }}</strong> users? This action can NOT be undone!",
+                "typeNumberToConfirm": "Please type in the number of users to confirm."
+            },
+            "deleted": "Deleted {{ param }} Users"
```

Comment:
Do you even use this anywhere?

Reply:
Yes.

Reply:
![image](https://user-images.githubusercontent.com/60005702/175774802-45bc2d86-3271-4825-87f8-0150008fb97d.png)